Product models (#72)
akeneo/onboarder-helpcenter: content/md/retailer-onboarder/review-suppliers-propositions/review-product-changes.md

---

# How to manage suppliers proposals?
## Products

> When product is completed on the Supplier Onboarder (ie [all attributes defined as required for the channel Onboarder](/onboarder/articles/define-product-requirements.html) are filled in by the supplier), it is sent as a proposal to your PIM.

Reviewer: ```suggestion
When a product is completed on the Supplier Onboarder (ie [all attributes defined as required for the channel Onboarder](/onboarder/articles/define-product-requirements.html) are filled in by the supplier), you receive a proposal in your PIM.
```

:::info
A product updated by one of your supplier will never impact directly your PIM. A review step is mandatory so that you can evaluate the proposition.
:::

You can review the suggestions made by suppliers in the [PIM proposals](https://help.akeneo.com/pim/serenity/articles/review-products-proposals.html).

## Product models with variants
[Product models with variants](https://help.akeneo.com/pim/serenity/articles/what-about-products-variants.html) are specific type of products you can receive proposals for.

> You can receive proposals in two different cases :

Reviewer: ```suggestion
You can receive proposals in two different cases:
```

1. for the product model values
1. for the product variant values

For the product model values, you receive a proposal:

> * the first time the supplier completes a product variant with its parents product models,

Reviewer: ```suggestion
* the first time the supplier completes a product variant with its parent product model,
```

* and then each time a product model is updated.

> For the product variant values, you receive a proposal each time a supplier complete or update a product variant.

Reviewer: ```suggestion
For the product variant values, you receive a proposal each time a supplier completes or updates a product variant.
```

## Assets

> New assets suggested by suppliers are included in the proposals and they are automatically added to the right asset collection when the new product is accepted.

Reviewer: ```suggestion
When new assets are suggested by suppliers, they are included in the proposals and automatically added to the right asset collection when the new product is accepted.
```

> You can [change the asset](https://help.akeneo.com/pim/serenity/articles/manage-asset-families.html) (label, other attributes).

Reviewer: ```suggestion

```
